chapters/usage-profile.md

```diff
@@ -0,0 +1,243 @@
+
+<br>
+
+### 31.1.2 Intent
+
+To describe detail intended usage of the package assumed by the package supplier in a manner that supplements those conditions contained in the software.
+
+### 31.1.3 Examples
+
+EXAMPLE 1 Tag: `UsageScope:`
+
+```text
+UsageScope: SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+```
+
+EXAMPLE 2 RDF: Property `spdx:UsageScope` in any `spdx:SpdxDocument`, `spdx:Package` or `spdx:File`
+
+```text
+    <UsageScope>
+      <UsageScopeElement>
+        <File rdf:about="http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82..."
+      </UsageScopeElement>
+      <Usage>SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+      </Usage>
+    </UsageScope>
+  ...
+
+```
+
+## 31.2 Usage Scope Comment field <a name="31.2"></a>
+
+### 31.2.1 Description
+
+This field provides a place for the SPDX document creator to record any general comments about the "usage scope".
+
+Table 31.2 — Metadata for the usage scope comment field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | Free form text that may span multiple lines, refers only to the immediately preceding usagescope.  |
+
+
+### 31.2.2 Intent
+
+To provide additional context. For example, describing prerequisite assumptions of usage scope by the package supplier.
+
+### 31.2.3 Examples
+
+EXAMPLE 1 Tag: `UsageScopeComment:`
+
+```text
+UsageScope: SPDXRef-a_specific_binary_verNNN: shipped for consumer product/ medical system / automotive system
+UsageScopeComment: A_specific_binary_verNNN is compiled and velified as production level quality
+```
+
+EXAMPLE 2 RDF: Property `rdfs:comment` in class `spdx:UsageScope`
+
+```text
+<UsageScope>
+         <rdfs:comment> A_specific_binary_verNNN is compiled and velified as production level quality</rdfs:comment>
+</UsageScope>
+```
+
+
+## 31.3 UsageScope SPDX identifier field <a name="31.3"></a>
+
+### 31.3.1 Description
+
+Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the SPDX identifier field is shown in Table NN.
+
+**Table 31.3 — Metadata for the UsageScope SPDX identifier field**
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | Yes |
```

Comment:
Adding new _required_ attributes is an incompatible change and should not be in a 2.x specification version

Reply:
Thank you for pointing out it. I've changed Required attribute of UsageProfile SPDX ID to "No".